sass/_controlbox.scss

```diff
@@ -303,6 +303,7 @@
 
         .controlbox-panes {
             background-color: var(--controlbox-pane-background-color);
+            border-right: 3px solid var(--chatroom-head-bg-color);
```

Comment:
You can't put a border to the right of the controlbox pane. What if the user opens a single-user chat, which is green? Then you'll see an orange border for a green chat.